source/frontend/src/features/properties/list/columns.tsx

```diff
@@ -73,4 +79,49 @@ export const columns = ({ municipalities }: Props): ColumnWithProps<IProperty>[]
       },
     },
   },
+  {
+    Header: '',
+    accessor: 'controls' as any, // this column is not part of the data model
+    align: 'right',
+    sortable: false,
+    width: 20,
+    Cell: (cellProps: CellProps<IProperty, number>) => {
+      const { hasClaim } = useKeycloakWrapper();
+
+      return (
+        <StyledDiv>
+          {hasClaim(Claims.PROPERTY_VIEW) && (
+            <StyledIconButton
+              data-testid="view-prop-tab"
+              title="View Property Info"
+              variant="light"
+            >
+              <Link
+                to={`/mapview/sidebar/property/${cellProps.row.original.id}?pid=${cellProps.row.original.pid}`}
```

Comment:
@Maheshbabu28 this isn't working currently as the side bar is expecting a pid that does not contain '-' characters.
Either update the sidebar code to format the pid as necessary or update your code to send the pid in the expected format.

Reply:
Fixed